src/cursor/cur_stat.c

```diff
@@ -249,6 +251,8 @@ __curstat_prev(WT_CURSOR *cursor)
 	if (cst->notpositioned) {
 		cst->notpositioned = false;
 		cst->key = WT_STAT_KEY_MAX(cst);
+		if (cst->next_set != NULL)
+			WT_ERR((*cst->next_set)(session, cst, false, true));
```

Comment:
@ddanderson, the `()` around `(*cst->next_set)` looks a little odd to me, stylistically, but *shrug*.